Src/CSharpier/SyntaxPrinter/SyntaxNodePrinters/CommonForEachStatement.cs

```diff
@@ -6,16 +6,30 @@
 
 namespace CSharpier.SyntaxPrinter.SyntaxNodePrinters
 {
-    public static class ForEachStatement
+    public static class CommonForEachStatement
     {
-        public static Doc Print(ForEachStatementSyntax node)
+        public static Doc Print(CommonForEachStatementSyntax node)
         {
             var groupId = Guid.NewGuid().ToString();
 
             var leadingTrivia = node.AwaitKeyword.Kind() != SyntaxKind.None
                 ? Token.PrintLeadingTrivia(node.AwaitKeyword)
                 : Token.PrintLeadingTrivia(node.ForEachKeyword);
 
+            Doc variable = Doc.Null;
+            if (node is ForEachStatementSyntax forEachStatementSyntax)
```

Comment:
So, you don't have to drink the kool-aid if you don't want to this time, but how would this look if the whole thing were an inline switch expression?

Reply:
better!